src/chigame/games/views.py

```diff
@@ -148,20 +148,35 @@ class TournamentCreateView(CreateView):
         "rules",
         "draw_rules",
         "num_winner",
-        "matches",
         "players",
     ]
-    # Note: "winner" is not included in the fields because it is not
-    # supposed to be set by the user. It will be set automatically
-    # when the tournament is over.
-    # Note: we may remove the "matches" field later for the same reason,
-    # but we keep it for now because it is convenient for testing.
+
+    def form_valid(self, form):
+        user = self.request.user
+        if user.tokens < 1:
```

Comment:
It seems like you are removing tokens for staff as well which makes it so staff can only create one tournament. I'm getting the same error regardless of whether I am creating a tournament from a staff account or not. Also, I'd prefer if you had the redirection before the user tries to fill our the form. Maybe you could do if user.tokens and user.is_staff and return redirect immediately.